sphinx/application.py

```diff
@@ -737,6 +737,12 @@ def add_directive(self, name, obj, content=None, arguments=None, **options):
         directive = directive_helper(obj, content, arguments, **options)
         directives.register_directive(name, directive)
 
+        from docutils.parsers.rst import Directive
```

Comment:
Can you import this at the top-level, please

Reply:
done